requirements.txt

```diff
@@ -0,0 +1,228 @@
+ipykernel==5.1.1
+ipython==7.6.1
+ipython-genutils==0.2.0
+ipywidgets==7.5.0
+isort==4.3.21
+itsdangerous==1.1.0
+jdcal==1.4.1
+jedi==0.13.3
+jeepney==0.4
+Jinja2==2.10.1
+joblib==0.13.2
+json5==0.8.4
+jsonschema==3.0.1
+jupyter==1.0.0
+jupyter-client==5.3.1
+jupyter-console==6.0.0
+jupyter-core==4.5.0
+jupyterlab==1.0.2
+jupyterlab-server==1.0.0
+keyring==18.0.0
+kiwisolver==1.1.0
+lazy-object-proxy==1.4.1
+libarchive-c==2.8
+lief==0.9.0
+llvmlite==0.29.0
+locket==0.2.0
+lxml==4.3.4
+MarkupSafe==1.1.1
+matplotlib==3.1.0
+mccabe==0.6.1
+minio==5.0.7
+mistune==0.8.4
+mkl-fft==1.0.12
+mkl-random==1.0.2
+mkl-service==2.0.2
+mock==3.0.5
+more-itertools==7.0.0
+mpmath==1.1.0
+msgpack==0.6.1
+multipledispatch==0.6.0
+navigator-updater==0.2.1
+nbconvert==5.5.0
+nbformat==4.4.0
+neo==0.8.0
+networkx==2.3
+nltk==3.4.4
+nose==1.3.7
+notebook==6.0.0
+numba==0.44.1
+numexpr==2.6.9
+numpy==1.16.4
+numpydoc==0.9.1
+odo==0.5.1
+olefile==0.46
+openpyxl==2.6.2
+packaging==19.0
+pandas==0.24.2
+pandocfilters==1.4.2
+parso==0.5.0
+partd==1.0.0
+path.py==12.0.1
+pathlib2==2.3.4
+patsy==0.5.1
+pep8==1.7.1
+pexpect==4.7.0
+pickleshare==0.7.5
+Pillow==6.1.0
+pkginfo==1.5.0.1
+pluggy==0.12.0
+ply==3.11
+progress==1.5
+progressbar2==3.53.1
+prometheus-client==0.7.1
+prompt-toolkit==2.0.9
+psutil==5.6.3
+ptyprocess==0.6.0
+py==1.8.0
+pycodestyle==2.5.0
+pycosat==0.6.3
+pycparser==2.19
+pycrypto==2.6.1
+pycurl==7.43.0.3
+pydot==1.4.1
+pyflakes==2.1.1
+Pygments==2.4.2
+pylint==2.3.1
+PyMySQL==0.9.3
+pyodbc==4.0.26
+pyOpenSSL==19.0.0
+pyparsing==2.4.0
+PyQt5==5.12
+PyQt5-sip==4.19.19
+PyQtWebEngine==5.12
+pyrsistent==0.14.11
+PySocks==1.7.0
+pytest==5.0.1
+pytest-arraydiff==0.3
+pytest-astropy==0.5.0
+pytest-doctestplus==0.3.0
+pytest-openfiles==0.3.2
+pytest-remotedata==0.3.1
+python-dateutil==2.8.0
+python-dotenv==0.14.0
+python-utils==2.4.0
+pytz==2019.1
+PyWavelets==1.0.3
+PyYAML==5.1.1
+pyzmq==18.0.0
+QtAwesome==0.5.7
+qtconsole==4.5.1
+QtPy==1.8.0
+quantities==0.12.4
+requests==2.22.0
+rope==0.14.0
+ruamel-yaml==0.15.46
+scikit-image==0.15.0
+scikit-learn==0.21.2
+scipy==1.3.0
+seaborn==0.9.0
+SecretStorage==3.1.1
+Send2Trash==1.5.0
+simplegeneric==0.8.1
+singledispatch==3.4.0.3
+sip==4.19.8
+six==1.12.0
+snowballstemmer==1.9.0
+sortedcollections==1.1.2
+sortedcontainers==2.1.0
+soupsieve==1.8
+Sphinx==2.1.2
+sphinxcontrib-applehelp==1.0.1
+sphinxcontrib-devhelp==1.0.1
+sphinxcontrib-htmlhelp==1.0.2
+sphinxcontrib-jsmath==1.0.1
+sphinxcontrib-qthelp==1.0.2
+sphinxcontrib-serializinghtml==1.1.3
+sphinxcontrib-websupport==1.1.2
+spyder==3.3.6
+spyder-kernels==0.5.1
+SQLAlchemy==1.3.5
+statsmodels==0.10.0
+sympy==1.4
+tables==3.5.2
+tblib==1.4.0
+terminado==0.8.2
+testpath==0.4.2
+toolz==0.10.0
+tornado==6.0.3
+tqdm==4.32.1
+traitlets==4.3.2
+unicodecsv==0.14.1
+urllib3==1.24.2
+wcwidth==0.1.7
+webencodings==0.5.1
+Werkzeug==0.15.4
+widgetsnbextension==3.5.0
+wrapt==1.11.2
+wurlitzer==1.0.2
+xlrd==1.2.0
+XlsxWriter==1.1.8
+xlwt==1.3.0
+zict==1.0.0
+zipp==0.5.1
```

Comment:
For requirements, we usually don't list every package in the pip freeze, but just a few packages that are needed. For example, datajoint and your speedgoat packages. The packages used by datajoint will be installed when installing datajoint.